xml/cap_depl_install_production.xml

```diff
@@ -419,7 +419,7 @@ and they must be run in this order:
   </listitem>
   <listitem>
     <para>
-        Deploy &scf;
+        Deploy SCF
```

Comment:
Should this be some variant of `&productname;` ?

Reply:
No, because it is the name of the namespace. I'll take another look at it-- might be confusing.